docs/docs/policies.mdx

```diff
@@ -732,33 +707,32 @@ default restart, label `2` a greeting and `3` indicates goodbye.
 
 It's often useful to include a bit more history than just the current state
 when predicting an action. The `TrackerFeaturizer` iterates over tracker
-states and calls a `SingleStateFeaturizer` for each state. There are two
-different tracker featurizers:
+states and calls a `SingleStateFeaturizer` for each state to create numeric
+input features for a policy.
+The target labels correspond to bot actions or bot utterances
+represented as index of a corresponding action in the domain.
+
+There are two different tracker featurizers:
 
 #### 1. Full Dialogue
 
 `FullDialogueTrackerFeaturizer` creates numerical representation of
 stories to feed to a recurrent neural network where the whole dialogue
 is fed to a network and the gradient is backpropagated from all time steps.
-Therefore, `X` is an array of shape
-`(num_stories, max_dialogue_length, num_input_features)` and
-`y` is an array of shape
-`(num_stories, max_dialogue_length, num_bot_features)`.
-The smaller dialogues are padded with `-1` for all features, indicating
+The smaller dialogues are padded with `0` for all features, indicating
 no values for a policy.
 
 #### 2. Max History
 
 `MaxHistoryTrackerFeaturizer` creates an array of previous tracker
-states for each bot action or utterance, with the parameter
-`max_history` defining how many states go into each row in `X`.
+states for each bot action or bot utterance, with the parameter
+`max_history` defining how many states go into each row of input features.
+If `max_history` is not specified, the algorithm takes into account
+the whole length of a dialogue.
+The smaller dialogues are padded with `0` for all features, indicating
+no values for a policy.
```

Comment:
"no values" sounds strange. Maybe something like "indicating that we don't have any values in this case as the dialogue does not has so many turns."

Reply:
I think simply removing it is ok